docs/tutorials/gallery_adding_an_assistant.py

```diff
@@ -0,0 +1,177 @@
+"""
+# Adding an Assistant
+
+Ragna has builtin support for [several assistants][ragna.assistants], but there may be cases where you 
+want to use one that is not currently supported.
+
+This tutorial walks you through the basics of creating an assistant for LLMs that are not
+currently supported. 
+
+!!! note
+
+    This tutorial assumes that our TutorialAssistant class (shown below) is located in the
+    file ragna/assistants/_tutorial.py.
```

Comment:
We cannot assume that. The point of this tutorial is to explain how users can implement their own assistant without forking Ragna and putting their stuff into the source code.